src/util/metaData.ts

```diff
@@ -0,0 +1,39 @@
+import AjvModule from 'ajv'
+
+const Ajv = AjvModule.default
+const ajv = new Ajv({ allErrors: true })
+
+const schema = {
+  type: 'object',
+  properties: {
+    Creator: { type: 'string', minLength: 1 },
+    Description: { type: 'string', maxLength: 80, minLength: 1 },
+    Language: { type: 'string', minLength: 1 },
+    Publisher: { type: 'string', minLength: 1 },
+    Title: { type: 'string', maxLength: 30, minLength: 1 },
+  },
+  required: ['Creator', 'Description', 'Language', 'Publisher', 'Title'],
+  additionalProperties: true,
+}
+
+const validate = ajv.compile(schema)
+
+export const validateMetadata = (metaData) => {
+  const valid = validate(metaData)
```

Comment:
I think wording is `sanitize` not `validate`, not great to create inconsistency in the vocabulary.

Reply:
In this case, we only validate the data with some rules but sanitizing implies that data will be modified with some rules. So I propose not renaming the function.